.github/workflows/prod-workflow.yml

```
@@ -193,57 +193,56 @@ jobs:
-  #     - name: Deploy APIs
-  #       env:
-  #         REGION: ap-northeast-2
-  #         ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-  #         SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  fetcher:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packages/fetcher
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)